uarch/uarch-machine-state-access.h

```diff
@@ -541,7 +541,15 @@ class uarch_machine_state_access : public i_state_access<uarch_machine_state_acc
         *pval = raw_read_memory<T>(paddr);
     }
 
-    void do_write_memory(uint64_t paddr, const unsigned char *data, uint64_t log2_size) {}
+    bool do_read_memory(uint64_t paddr, unsigned char *data, uint64_t length) {
```

Comment:
Should it be ever used?

Reply:
This is not triggered in uarch, just needed to compile.